Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [scripts]
 lint = "flake8 ."
-publish = "python setup.py sdist upload"
+publish = "python setup.py sdist upload -r wizeline"
```

Comment:
Relayer is a public package ([See pypi.org](https://pypi.org/project/relayer/)).

Reply:
Nice catch!